packages/dds/merge-tree/src/client.ts

```diff
@@ -716,7 +715,7 @@ export class Client extends TypedEventEmitter<IClientEvents> {
      * @param segment - The segment to find the position for
      * @param localSeq - The localSeq to find the position of the segment at
      */
-    protected findReconnectionPosition(segment: ISegment, localSeq: number) {
+    public findReconnectionPosition(segment: ISegment, localSeq: number) {
```

Comment:
IMO you should mark this @internal. consumers outside this repo shouldn't be calling it.